client/src/resources/ItemActionController.js

```diff
@@ -0,0 +1,109 @@
+import { inject } from "aurelia-framework";
+import { Router } from "aurelia-router";
+import { Notification } from "aurelia-notification";
+import { I18N } from "aurelia-i18n";
+import { DialogService } from "aurelia-dialog";
+import { ConfirmDialog } from "dialogs/confirm-dialog.js";
+import QConfig from "resources/QConfig.js";
+
+@inject(Router, Notification, I18N, DialogService, QConfig)
+export default class ItemActionController {
+  constructor(router, notification, i18n, dialogService, qConfig) {
+    this.router = router;
+    this.notification = notification;
+    this.i18n = i18n;
+    this.dialogService = dialogService;
+    this.qConfig = qConfig;
+  }
+
+  activate(item) {
+    item.activate();
+  }
+
+  deactivate(item) {
+    item.deactivate();
```

Comment:
explicitly return the result of item.deactivate here

Reply:
implemented in -> https://github.com/nzzdev/Q-editor/pull/133/commits/1158ee8e70ecb7f0adf1542b666047acda0cfca7